### PR TITLE
Add flasgger recipe

### PR DESCRIPTION
**Pull request by lvoliveira**

### CODE REVIEW

**Comment by lvoliveira** [comment]
Will this branch be merged?

File: recipes/flasgger/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (nicoddemus) [inline recipes/flasgger/meta.yaml]: Is this package pure-python? If so, please add:

```yml
build:
  noarch: python
```

So this becomes a pure python package.

File: recipes/flasgger/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+about:
+  home: https://github.com/rochacbruno/flasgger
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Easy Swagger UI for your Flask API'
+
+extra:
+  recipe-maintainers:
+    - lvoliveira
```
Comment (nicoddemus) [inline recipes/flasgger/meta.yaml]: Please add mu and @igortg to this list as well. 😁

File: recipes/flasgger/meta.yaml (hunk 1 above)
Comment (lvoliveira) [reply recipes/flasgger/meta.yaml]: Done

File: recipes/flasgger/meta.yaml (hunk 2 above)
Comment (lvoliveira) [reply recipes/flasgger/meta.yaml]: Done

**Comment by nicoddemus** [comment]
@conda-forge/staged-recipes this is a `noarch` package and it has already passed on CircleCI so I believe this is ready to review/merge. 👍

**Comment by nicoddemus** [comment]
Thanks @ocefpaf!

**Pull request merged**